package.json

```diff
@@ -1,18 +1,16 @@
 {
```

Comment:
about the frontend packages update, if you run into some merge conflicts, always use the incoming changes from master, some package updates here could lead to faulty builds.

Reply:
only exceptions are the scripts variables

Reply:
sry